mlir/test/Conversion/VectorToArmSME/vector-to-arm-sme.mlir

```diff
@@ -162,6 +162,21 @@ func.func @fold_transpose_into_load(%src : memref<?x?xf32>) {
 
 // -----
 
+// CHECK-LABEL: @fold_transpose_into_load_multi_use
+// CHECK: arm_sme.tile_load {{.*}} : memref<?x?xf32>, vector<[4]x[4]xf32>
+// CHECK: arm_sme.tile_store {{.*}} : memref<?x?xf32>, vector<[4]x[4]xf32>
+// CHECK: arm_sme.tile_load {{.*}} layout<vertical> : memref<?x?xf32>, vector<[4]x[4]xf32>
```

Comment:
Would you mind adding more Ops? I'm actually curious to see the final "prevent.dce" Op. Also, somewhat tangential, we should probably switch to `"test.some_use"`.

Reply:
> Would you mind adding more Ops? I'm actually curious to see the final "prevent.dce" Op.

👍 

> Also, somewhat tangential, we should probably switch to `"test.some_use"`.


I agree, still a few tests where we're using `prevent.dce` that need updating. I added the single `test.some_use` here as `prevent.dce` is misleading.